ext/opentelemetry-ext-mysql/tests/test_mysql_integration.py

```diff
@@ -16,20 +16,44 @@
 
 import mysql.connector
 
-from opentelemetry.ext.mysql import trace_integration
+import opentelemetry.ext.mysql
+from opentelemetry.sdk import resources
 from opentelemetry.test.test_base import TestBase
 
 
 class TestMysqlIntegration(TestBase):
     def test_trace_integration(self):
-        tracer = self.tracer_provider.get_tracer(__name__)
+        with mock.patch("mysql.connector.connect") as mock_connect:
+            mock_connect.get.side_effect = mysql.connector.MySQLConnection()
+            opentelemetry.ext.mysql.trace_integration()
+            cnx = mysql.connector.connect(database="test")
+            cursor = cnx.cursor()
+            query = "SELECT * FROM test"
+            cursor.execute(query)
+
+        spans_list = self.memory_exporter.get_finished_spans()
+        self.assertEqual(len(spans_list), 1)
+        span = spans_list[0]
+        # TODO: Add more tests?
```

Comment:
What kind of tests are missing?

Reply:
None, I added this in other commit and somehow it ended here.